data/report_templates/project_summary.md

```diff
@@ -132,10 +132,10 @@ No lanes information to be displayed.
 {% else %}
 Date | Flowcell | Lane | Clusters(M) | >=Q30(%) | PhiX | Method
 :-----|:----------|:------|:-------------|:----------|:------|:-------
-{%- for fc in project.flowcells.values()|sort(attribute='date') %}
-{%- for lane in fc.lanes.values() %}
+{% for fc in project.flowcells.values()|sort(attribute='date') -%}
+{% for lane in fc.lanes.values() -%}
 {{ fc.date }} | `{{ fc.name }}` | {{ lane.id }} | {{ lane.total_reads_proj }} | {{ lane.weighted_avg_qval_proj }} | {{ lane.phix }} | Seq. {{ fc.seq_meth }}
-{%- endfor %}
+{% endfor -%}
 {% endfor %}
```

Comment:
Looking at old code as you said, the last endfor should probably be
```suggestion
{% endfor -%}
{%- endfor %}
```

Reply:
I've made the changes on the latest stage if you want to check (including the last endfor)